### DISCUSSION TITLE
Chronological Timeline

### DISCUSSION THREAD

**Discussion by DaB-Dot** [tateisu/SubwayTooter#202] [Q&A]
Hi
Is there a chance to start any timeline at the last read toot instead of the most recent? I want to scroll forward in the timeline not in backwards.
Is there a setting i can activate? Otherwise I like to make it a feature request.
Thank you! Kind regards

**Comment by tateisu** [comment]
Mastodon server saves latest 400 posts for each home and list TL.
Older posts is exists in server's database, but not in home and list TL.
When you reopen the app, the client can read latest 400 posts from server.
This is a server-side limitation.

In this situation If the top refresh is switched from the continuation of the current reading range , the application can't detect the gap of missing posts. (cannot display as a gap).
